<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiscreteGaussianFilterInterface</class>
 <widget class="QWidget" name="DiscreteGaussianFilterInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>213</width>
    <height>207</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget_7">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>50</y>
     <width>205</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="varianceHLayout_bottom"/>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>4</y>
     <width>207</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="activeHLayout" stretch="1,2">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="activeLabel">
      <property name="text">
       <string>Target:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="activeComboBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_8">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>29</y>
     <width>205</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="varianceHLayout_top">
    <item>
     <widget class="QLabel" name="varianceLabel">
      <property name="text">
       <string>Variance</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="varianceHSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_9">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>100</y>
     <width>205</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="kernalHLayout_top">
    <item>
     <widget class="QLabel" name="kernalLabel">
      <property name="text">
       <string>Maximum Kernal Width</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="kernalHSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_10">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>121</y>
     <width>205</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="kernalHLayout_bottom"/>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>171</y>
     <width>205</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="replaceInvertLayout">
    <item>
     <widget class="QCheckBox" name="replaceCheckBox">
      <property name="text">
       <string>Replace</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="invertButton">
      <property name="text">
       <string>Invert</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
